src/tests/test_definitions.py

```diff
@@ -262,16 +262,30 @@ def test_pep257():
         'capitalization',
         'comment_after_def_bug',
     )
-    for test_case in test_cases:
-        case_module = __import__('test_cases.{0}'.format(test_case),
-                                 globals=globals(),
-                                 locals=locals(),
-                                 fromlist=['expectation'],
-                                 level=1)
-        results = list(check([os.path.join(os.path.dirname(__file__),
-                                           'test_cases', test_case + '.py')],
-                             select=set(ErrorRegistry.get_error_codes())))
-        for error in results:
-            assert isinstance(error, Error)
-        results = set([(e.definition.name, e.message) for e in results])
-        assert case_module.expectation.expected == results
+
+    option_test_cases = (
+        'multi_line_summary_start',
+    )
+
+    optional_codes = set(('D212', 'D213'))
+
+    configurations = (
+        (test_cases, set(ErrorRegistry.get_error_codes()) - optional_codes),
+        (option_test_cases, optional_codes)
+    )
+
```

Comment:
Instead of all of this, you can just configure the test cases to always run on all the errors. It's okay if we'll get mutually exclusive errors in these and it will be less of a hassle.